packages/features/ee/managed-event-types/lib/handleChildrenEventTypes.ts

```diff
@@ -146,17 +141,6 @@ export default async function handleChildrenEventTypes({
   // Calculate if there are new workflows for which assigned members will get too
   const currentWorkflowIds = eventType.workflows?.map((wf) => wf.workflowId);
 
-  // Define hashedLink query input
-  const hashedLinkQuery = (userId: number) => {
-    return hashedLink
-      ? !connectedLink
-        ? { create: { link: generateHashedLink(userId) } }
-        : undefined
-      : connectedLink
-      ? { delete: true }
-      : undefined;
-  };
-
```

Comment:
Why is this removed?

Reply:
@Amit91848 This was responsible for updating the hashedLink(s) for child event types for "managed events".

But but, we are not showing/allowing the user to edit single-use links from the parent event type, i.e the parent managed event.

So, do you think this logic to update the single-use-links of the child events is needed?

Please let me know if I am missing something. 🙏